test/team_stats_test.rb

```diff
@@ -0,0 +1,178 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+require './lib/stat_parser'
```

Comment:
One thing I did was to require 'stat_parser' in the 'stat_tracker' class -- so it should not be necessary to require it here